### PR TITLE
Ring install lib centos6 support

### PR DESCRIPTION
**Pull request by cazino**
I think this is ready for merging.

### CODE REVIEW

File: jenkins/ring-install.sh [hunk 1]
```diff
@@ -8,7 +22,44 @@ test -n "${HOST_IP:-}" || (echo "HOST_IP should be defined." && return 1);
...
+    echo "Nothing specific here."
+}
+
 function add_source {
+    distro_dispatch add_source_centos add_source_ubuntu
+}
+
+function add_source_centos {
+    set +x
+    sudo sh -c "cat <<-EOF >/etc/yum.repos.d/scality.repo
```
Comment (JordanP) [inline jenkins/ring-install.sh:52]: use `tee` instead

Comment (JordanP) [inline jenkins/ring-install.sh:52]: It was more a comment then a change request. Am fine with the current code

File: jenkins/ring-install.sh [hunk 2]
```diff
@@ -78,9 +139,43 @@ EOF
...
+    "name": "node-n",
+    "nodes": "1",
+    "ip": "$HOST_IP",
+    "chord-ip": "$HOST_IP",
+    "supervisor-ip": "$HOST_IP",
+    "ssl": "0",
+    "tier2": false
+}
+EOF
+    local scnc_path=$(which scality-node-config)
```
Comment (JordanP) [inline jenkins/ring-install.sh:161]: Why do you have to do that ? Maybe leave a code comment

File: jenkins/distro-utils.sh [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    [ "$os_VENDOR" = "Ubuntu" ]
+}
+
+function distro_dispatch {
+    if is_centos ; then
+        $1
+    elif is_ubuntu; then
+        $2
+    else
+        echo "Unkown OS"
```
Comment (JordanP) [inline jenkins/distro-utils.sh]: Typo: Unknown

File: jenkins/ring-install.sh [hunk 4]
```diff
@@ -161,28 +295,52 @@ function install_sproxyd {
...
+    sudo /etc/init.d/scality-sproxyd restart
+    sudo /usr/local/scality-sagentd/sagentd-manageconf -c /etc/sagentd.yaml add `hostname -s`-sproxyd type=sproxyd ssl=0 port=10000 address=$HOST_IP path=/run/scality/connectors/sproxyd
+    sudo /etc/init.d/scality-sagentd restart
+}
 
+function install_sproxyd_centos {
+    sudo yum install -y scality-sproxyd-httpd
+    # https://docs.scality.com/display/R43/Install+sproxyd+on+CentOS+or+RedHat
+    sudo sed -i "s/^#LoadModule fastcgi_module modules\/mod_fastcgi.so/LoadModule fastcgi_module modules\/mod_fastcgi.so/" /etc/httpd/conf.d/fastcgi.conf
+    #sudo sed -i "s/^Include conf.d\/*.conf/User apache\nGroup apache\nInclude conf.d\/*.conf/" /etc/httpd/conf/httpd.conf
```
Comment (JordanP) [inline jenkins/ring-install.sh]: Remove this line

**Comment by JordanP** [comment]
Once your done, you should squash all the ~50 commits into one.

File: jenkins/ring-install.sh (hunk 2 above)
Comment (cazino) [inline jenkins/ring-install.sh:161]: I did, just above the first line, so it is not showing up here.

**Comment by JordanP** [comment]
LGTM

**Pull request merged**